<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserWindow</class>
 <widget class="QMainWindow" name="BrowserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BrowserWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="ButtonsAndUrl" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="backButton">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshButton">
         <property name="text">
          <string>Refresh</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="forwardButton">
         <property name="text">
          <string>Forward</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="urlEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="goButton">
         <property name="text">
          <string>Go</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="tabButton">
         <property name="text">
          <string>Add Tab</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="Bookmarks" native="true">
      <layout class="QHBoxLayout" name="bookmarksLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
